src/server.c

```diff
@@ -5868,7 +5869,7 @@ sds genRedisInfoString(dict *section_dict, int all_sections, int everything) {
             "keyspace_misses:%lld\r\n", server.stat_keyspace_misses,
             "pubsub_channels:%ld\r\n", dictSize(server.pubsub_channels),
             "pubsub_patterns:%lu\r\n", dictSize(server.pubsub_patterns),
-            "pubsubshard_channels:%lu\r\n", dictSize(server.pubsubshard_channels),
+            "pubsubshard_channels:%llu\r\n", server.shard_channel_count,
```

Comment:
If dictSize returns a `long`, then why make this `long long`? Does this PR imply we support more than `long` channels in platforms where long long is bigger than long?

Reply:
`server.shard_channel_count` is the sum of all 16384 dictSize, so I made it `long long` just to imply that this might be bigger than `long`.